src/applications/check-in/utils/appointment/index.js

```diff
@@ -72,8 +72,18 @@ const removeTimeZone = payload => {
 
   return updatedPayload;
 };
+
+const preCheckinExpired = appointments => {
```

Comment:
sorry for the drive-by comment, I'm realizing as I look at my own branch  that we need to make sure to add unit tests for any new functions here :-)